nomos-services/api/src/http/storage.rs

```diff
@@ -1,17 +1,19 @@
 use nomos_core::{block::Block, da::blob::Blob, header::HeaderId};
-use nomos_da_storage::rocksdb::{key_bytes, DA_SHARED_COMMITMENTS_PREFIX};
+use nomos_da_storage::rocksdb::{
+    create_blob_idx, key_bytes, DA_BLOB_PREFIX, DA_SHARED_COMMITMENTS_PREFIX,
+};
 use nomos_storage::{
     backends::{rocksdb::RocksBackend, StorageSerde},
     StorageMsg, StorageService,
 };
-use serde::de::DeserializeOwned;
+use serde::{de::DeserializeOwned, Serialize};
 
 pub async fn block_req<S, Tx>(
     handle: &overwatch_rs::overwatch::handle::OverwatchHandle,
     id: HeaderId,
 ) -> Result<Option<Block<Tx, full_replication::BlobInfo>>, super::DynError>
```

Comment:
This should be `kzgrs-backend::dispersal::BlobInfo`, the types are identical, but slipped when moving from fullreplication to kzgrs. Could you double check if we use full replication anywhere else in the http api?